### PR TITLE
Adding fetch for items

### PR DESCRIPTION
**Pull request by sievertknorrblad**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-                <Route path="/item/:itemId" render={(props) => <NewsItem id={props.match.params.itemId} /> } />
-              </Switch>
-            </Content>
-            <Footer />
-          </Page>
+        <Page>
+          <Header />
+          <Content>
+            <Switch>
+              <Route exact path="/" render={() => <PageNewsList />} />
```
Comment (iamstarkov) [inline src/App.js]: use `component` prop instead

File: src/utils/api.js [hunk 2]
```diff
@@ -7,112 +7,19 @@
...
-    "title": "'Anti-authority' tech rebels take on ISPs, connect NYC with cheap Wi-Fi",
-    "type": "story",
-    "url": "http://www.cbc.ca/news/technology/wifi-nyc-mesh-new-york-city-1.4617106"
-  }
+export const getItemsIds = () => {
+  return fetchJson('https://hacker-news.firebaseio.com/v0/topstories.json')
+  .then(data => {
+    return Promise.resolve(first10(data))
+  })
 };
```
Comment (iamstarkov) [inline src/utils/api.js]: how about this?

```diff
-export const getItemsIds = () => {
-   return fetchJson('https://hacker-news.firebaseio.com/v0/topstories.json')
-    .then(data => {
-      return Promise.resolve(first10(data))
-    })
-};

+const BASE = `https://hacker-news.firebaseio.com`;
+export const getItemsIds = () => {
+  return fetchJson(`${BASE}/v0/topstories.json`)
+    .then(data => first10(data))
+};
```

Comment (iamstarkov) [reply src/utils/api.js]: you won't believe what is written in promises' documentation.

11/10  would recommend

Comment (iamstarkov) [reply src/utils/api.js]: I mean your code works, but its verbose. you can simplify it how I did it. but in order to understand it you need to read Promise docs on MDN

File: src/App.js (hunk 1 above)
Comment (iamstarkov) [reply src/App.js]: as react-router documentation tells us

**Review by iamstarkov — APPROVED** [review]
there are few minor issues, which I would like to be fixed before you merge

LGTM

File: src/utils/api.js (hunk 2 above)
Comment (sievertknorrblad) [reply src/utils/api.js]: Oh, I did this at first but had issues with it. Works now :)

**Comment by iamstarkov** [comment]
@sievertknorrblad merge it

**Pull request merged**